Boeing-Ontology/ontology/Boeing.sadl

```diff
@@ -23,7 +23,10 @@ SubDD_Req (note "A Requirement identified in the Subsystem Design Document") is
 	Rq:satisfies of SubDD_Req only has values of type SRS_Req.
 
 SBVT_Test (note "A test identified in the Software Baseline Verification Tests") is a type of TEST.
+   verifies of SBVT_Test_Procedure only has values of type {SubDD_Req or SRS_Req}.
```

Comment:
@kquick, can we add CSID_Req to the list of verifies? There are instances of these in the software test report.